integration_tests/pages/match/searchPage.ts

```diff
@@ -22,18 +20,6 @@ export default class SearchPage extends Page {
     this.shouldShowMatchingDetails(placementRequest)
   }
 
-  shouldShowKeyPersonDetails(placementRequest: PlacementRequestDetail) {
-    cy.get('.prisoner-info').within(() => {
-      const { person } = placementRequest
-      if (!isFullPerson(person)) throw new Error('test requires a Full Person')
-
-      cy.get('span').contains(person.name)
-      cy.get('span').contains(`CRN: ${person.crn}`)
-      cy.get('span').contains(`Tier: ${placementRequest?.risks?.tier?.value.level}`)
-      cy.get('span').contains(`Date of birth: ${DateFormats.isoDateToUIDate(person.dateOfBirth, { format: 'short' })}`)
-    })
-  }
```

Comment:
Does the search page not contain this component anymore? I didn't see any changes to the search page only to a show page